CONTRIBUTING.md

```diff
@@ -1,7 +1,332 @@
-To contribute to the Void documentation, please read the following sections of the Void Handbook. Pull requests that do not meet the criteria described in these sections will not be merged. Please try to address as many of the criteria as possible before submitting a PR - that will improve the chances of your contribution being accepted quickly.
+# Contributing
 
-- [About this Handbook](https://docs.voidlinux.org/about/about-this-handbook.html)
-- [Style Guide](https://docs.voidlinux.org/contributing/void-docs/style-guide.html)
-- [Submitting Changes](https://docs.voidlinux.org/contributing/void-docs/submitting.html)
+To contribute to the Void documentation, please read the following. Pull
+requests that do not meet the criteria described below will not be merged.
+Before submitting a PR:
 
-In particular, please be sure to run this repository's `check.sh` script, and to address any issues it raises, before submitting your PR.
+- try to address as many of the criteria as possible; and
+- run the `check.sh` script provided in the repository root, addressing any
+   issues it reports.
+
+This will improve the chances of your contribution being accepted quickly.
+
+## Contents
+
+- [Suitable Content](#suitable-content)
+- [Style Guide](#style-guide)
+- [Submitting Changes](#submitting-changes)
+
+## Suitable content
+
+The Handbook is not intended to be a general guide to using a Linux system, as
+[noted in the
+Handbook](https://docs.voidlinux.org/about/about-this-handbook.html):
+
+> This handbook is not an extensive guide on how to use and configure common
+> Linux software. The purpose of this document is to explain how to install,
+> configure, and maintain Void Linux systems, and to highlight the differences
+> between common Linux distributions and Void ...
+> 
+> Those looking for tips and tricks on how to configure a Linux system in
+> general should consult upstream software documentation. Additionally, the Arch
+> Wiki provides a fairly comprehensive outline of common Linux software
+> configuration, and a variety of internet search engines are available for
+> further assistance.
+
+Thus, we are unlikely to accept contributions which add information that is not
+particularly Void-specific.
+
+## Style Guide
+
+This style guide outlines the standards for contributing to the
+[void-docs](https://github.com/void-linux/void-docs/) project. The manual on
```

Comment:
"The handbook" or "The documentation"

Reply:
i've gone with "the Handbook", since there's other documentation (e.g. XBPS man pages) which are essentially "Void documentation", but not covered by the style guide.